src/Types/Value.purs

```diff
@@ -156,7 +156,7 @@ unsafeAdaToken = TokenName mempty
 -- | exported
 mkTokenName :: ByteArray -> Maybe TokenName
 mkTokenName byteArr =
-  if byteLength byteArr > 0 then pure $ TokenName byteArr else Nothing
+  if byteLength byteArr <= 32 then pure $ TokenName byteArr else Nothing
```

Comment:
I might be completely mistaken about this, but are we going to end up with two different encodings of token names? CSL uses CBOR, but I don't think Ogmios does

Reply:
I'm not sure why would it be looking at that change. Both CSL and Ogmios limits the size of TokenName to 32 bytes. I'm not sure what do you mean.